extensions/smallrye-reactive-messaging-amqp/deployment/src/main/java/io/quarkus/smallrye/reactivemessaging/amqp/deployment/AmqpDevServicesProcessor.java

```diff
@@ -138,6 +129,16 @@ public DevServicesResultBuildItem startAmqpDevService(
             closeBuildItem.addCloseTask(closeTask, true);
         }
         cfg = configuration;
+
+        if (devService.isOwner()) {
+            Map<String, String> config = devService.getConfig();
+            log.infof("Dev Services for AMQP started. Other Quarkus applications in dev mode will find the "
+                    + "broker automatically. For Quarkus applications in production mode, you can connect to"
+                    + " this by starting your application with -Damqp.host=%s -Damqp.port=%s -Damqp.user=%s -Damqp.password=%s",
+                    config.get(AMQP_HOST_PROP), config.get(AMQP_PORT_PROP), config.get(AMQP_USER_PROP),
+                    config.get(AMQP_PASSWORD_PROP));
```

Comment:
I would prefer if we kept the message separate to be more consistent with the logging we do in other dev services (i.e. one line with the started message).

Reply:
not everywhere - https://github.com/quarkusio/quarkus/blob/main/extensions/kafka-client/deployment/src/main/java/io/quarkus/kafka/client/deployment/DevServicesKafkaProcessor.java#L129. I still think that having it in two messages is not connected at first sight. At least, it's not totally clear to me. But I'll change it back if you request.